Missing packages/function-tree/src/providers/Debugger.js

Just tried to run fresh `npm run setup` after `npm i` in monorepo and got the following error during build of todomvc:
```sh
> @cerebral/todomvc@0.0.0 prepublish /home/pacha/web/cerebral/cerebral/demos/todomvc
> npm run build



> @cerebral/todomvc@0.0.0 build /home/pacha/web/cerebral/cerebral/demos/todomvc
> react-scripts build

Creating an optimized production build...
Failed to compile.

Module not found: Error: Cannot resolve 'file' or 'directory' ./lib/providers/Debugger in /home/pacha/web/cereb
ral/cerebral/packages/function-tree


Errored while running BootstrapCommand.execute
Error: Command failed: npm run prepublish
```

Seems that `packages/function-tree/src/providers/Debugger.js` / `packages/function-tree/lib/providers/Debugger.js` are missing, but required in [packages/function-tree/providers.js](https://github.com/cerebral/cerebral/blob/master/packages/function-tree/providers.js#L2).

--- a/packages/function-tree/README.md
+++ b/packages/function-tree/README.md
@@ -443,55 +443,6 @@ const tree = [
 execute(tree, {foo: 'bar'})
 ```
 
-#### Debugger (optional provider)
-Download the [Chrome Extension](https://chrome.google.com/webstore/detail/function-tree-debugger/ppfbmcnapdgakfiocieggdgbhmlalgjp).
-
-```js
-import FunctionTree from 'function-tree'
-import {ContextProvider, DebuggerProvider} from 'function-tree/providers'
-import request from 'request'
-
-const execute = FunctionTree([
-  DebuggerProvider({
-    colors: {
-      request: 'red' // Set color of usage in debugger
-    }
-  }),
-  ContextProvider({
-    request
-  })
-])
-```
-
-When you execute trees you can name the execution:
-
-```js
-execute('thisHappened', someTree)
-```
-
-And that will be used in the debugger instead of execution id.
-
-#### NodeDebugger (optional provider)
-```js
-const FunctionTree = require('function-tree')
-const NodeDebuggerProvider = require('function-tree/providers/NodeDebugger')
-const ContextProvider = require('function-tree/providers/Context')
-const request = require('request')
-
-const execute = FunctionTree([
-  NodeDebuggerProvider({
-    colors: {
-      request: 'red' // Set color of usage in debugger
-    }
-  }),
-  ContextProvider({
-    request
-  })
-])
-```
-
-Colors supported are **red, green, yellow, blue, magenta, cyan, white, gray, bgBlack, bgRed, bgGreen, bgYellow, bgBlue, bgMagenta, bgCyan and bgWhite**
-
 #### Events
 The execute function is also an event emitter.
 

--- a/packages/function-tree/providers.js
+++ b/packages/function-tree/providers.js
@@ -1,7 +1,5 @@
 exports.ContextProvider = require('./lib/providers/Context').default
-exports.DebuggerProvider = require('./lib/providers/Debugger').default
 exports.ExecutionProvider = require('./lib/providers/Execution').default
 exports.PropsProvider = require('./lib/providers/Props').default
-exports.NodeDebuggerProvider = require('./lib/providers/NodeDebugger').default
 exports.PathProvider = require('./lib/providers/Path').default
 exports.ReduxProvider = require('./lib/providers/Redux').default

--- a/website/builder/package.json
+++ b/website/builder/package.json
@@ -7,7 +7,7 @@
   "scripts": {
     "start": "babel-watch builder/dev --presets es2015,react --watch docs pages css scripts",
     "build": "babel-node builder/build --presets es2015,react",
-    "prepublish": "ln -s ../../docs ./docs"
+    "prepublish": "ln -nfs ../../docs ./docs"
   },
   "author": "",
   "license": "ISC",

